site/content/integrate/apps/api/manifest/_index.md

```diff
@@ -0,0 +1,143 @@
+
+| Name              | Description                                                                  |
+| :---------------- | :--------------------------------------------------------------------------- |
+| `/post_menu`      | An item in the post menu.                                                    |
+| `/channel_header` | Webapp: A button in the channel header, Mobile: An item in the channel menu. |
+| `/command`        | A slash command.                                                             |
+| `/in_post`        | Attachment embedded to a post.                                               |
+
+### Deployment Information
+
+#### `http`: Apps accessible via HTTP
+
+Provides configuration for accessing an app that is deployed exterally of Mattermost and is accessible via HTTP.
+
+| Name                               | Description                                                                               |
+| :--------------------------------- | :---------------------------------------------------------------------------------------- |
+| `root_url` | Base URL to send all calls and static asset requests to. |
+| `use_jwt`  | Include a secret-based JWT in all requests to the App. The secret must be provided by the app to the sysadmin and entered at the app's installation time. |
+
+#### `aws_lambda`: Apps deployable and accessible as AWS Lambda functions and S3 static assets
+
+Provides configuration for deploying an app (bundle) onto the AWS Lambda and S3
+services and accessing it there.
+
+| Name          | Type                | Description |
+| :------------ | :------------------ | :---------------- |
+| `function`(*) | AWS Lambda Function | List of functions |
+
+Each function contains the following fields:
+
+| Name         | Description |
+| :----------- | :---------- |
+| `path`(*)    | (Root) path of calls to be mapped to this function, e.g. `"/"` |
+| `name`(*)    | Function's base name. Is used as a base name to compose the actual AWS Lambda name. `{name}.zip` is used looking up the lambda function in an app bundle by `appsctl aws deploy` |
+| `handler`(*) | The name of the handler go function |
+| `runtime`(*) | AWS Lambda runtime to use, e.g. `"go1.x"` |
+
+#### `open_faas`: Apps deployable and accessible as OpenFaaS or faasd functions
+
+Provides configuration for deploying an app (bundle) onto OpenFaaS and faasd platforms.
+
+| Name          | Type              | Description |
+| :------------ | :---------------- | :---------------- |
+| `function`(*) | OpenFaaS Function | List of functions |
+
+Each function contains the following fields:
+
+| Name      | Description |
+| :-------- | :---------- |
+| `path`(*) | (Root) path of calls to be mapped to this function, e.g. `"/"` |
+| `name`(*) | Function's base name. Is used as a base name to compose the actual OpenFaaS name, combined with the app ID and version. |
+
+#### `Kubeless`: Apps deployable and accessible as Kubeless
```

Comment:
Link to Kubless starting guide?

Reply:
My suggestion would be to just link to https://kubeless.io/docs/quick-start/ just to transfer basic knowledge about what Kubeless does.